docs/theme-reference.md

```diff
@@ -5,33 +5,62 @@ you therefore need to customize the default theme, create your own theme or use
 
 ## Contents
 
+* [Customize the default theme](#customize-the-default-theme)
+* [Create your own theme](#create-your-own-theme)
 * [Theme structure](#theme-structure)
+* [Theme configuration](#theme-configuration)
 * [Required templates](#required-templates)
 * [Custom templates](#custom-templates)
-* [Customize the default theme](#customize-the-default-theme)
-* [Create your own theme](#create-your-own-theme)
+* [Pre-build hooks](#pre-build-hooks)
+
+## Customize the default theme
+
+When you create a new project using `verless create project`, verless generates a default theme inside the `themes`
+directory.
+
+You can customize this theme by changing the templates and stylesheets. To view your changes, re-build the project or
+just use `verless serve -w my-blog`.
+
+## Create your own theme
+
+The simplest way to create your own theme is to use the `verless create theme` command. It expects two arguments: The
+project for which you want to create a theme and the theme name itself.
+
+For example, creating the `dark-theme` theme for the `my-blog` project is as easy as:
+
+```shell script
+$ verless create theme my-blog dark-theme
+```
+
+Or, if you already are _inside_ the `my-blog` directory:
+
+```shell script
+$ verless create theme . dark-theme
+```
```

Comment:
We don't need to pass `.` to this command as it is default project path. The command should be
`verless create theme dark-theme`

Reply:
You're right, this needs to be updated. Thanks 👍